upt_macports/upt_macports.py

```diff
@@ -258,7 +258,7 @@ def create_package(self, upt_pkg, output=None):
     def package_versions(self, name):
         try:
             port_name = self.pkg_classes[self.frontend](
-            )._normalized_macports_name(name)
+            )._normalized_macports_folder(name)
```

Comment:
So, can we avoid creating a new object here? Just like in PR #47 .

Reply:
Is it fine now?